common/api-review/firestore-lite.api.md

```diff
@@ -80,9 +80,12 @@ export function doc(reference: DocumentReference<unknown>, path: string, ...path
 
 // @public
 export interface DocumentData {
-    [field: string]: any;
+    [field: string]: DocumentFieldValue;
 }
 
+// @public
+export type DocumentFieldValue = any;
```

Comment:
Please move the declaration of `DocumentFieldValue` to immediately above `DocumentData`. Make this change here and elsewhere in this PR.